src/app/components/header/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Link = styled.a`
+  font-family: ReithSans, Arial, Helvetica, freesans, sans-serif;
+  font-size: 2.2em;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+`;
+
+const StyledHeader = styled.header`
+  background-color: #bb1919;
+  height: 40px;
+  width: 100%;
+  padding: 15px 16px;
+`;
+
+const Header = () => (
+  <StyledHeader role="banner" aria-label="BBC News">
```

Comment:
@jp-bbc if we only have one banner (e.g. there is no ORB) then we don't need an aria-label...

Reply:
@jp-bbc Does this output in a HTML header tag?

Reply:
@greenc05 Yes it does render a <header>

Reply:
@greenc05 I'll remove the aria-label